en/about/faq.md

```diff
@@ -0,0 +1,43 @@
+# Frequently Asked Questions (FAQ)
+
+## Users
+
+<dl>
+  <dt>How efficient is MAVLink?</dt>
+  <dd>MAVLink is a very efficient protocol. It has just 8 bytes overhead per packet, including start sign and packet drop detection.</dd>
+
+  <dt>How many vehicles does MAVLink support?</dt>
+  <dd>255 vehicles, with system IDs ranging from 1 to 255 (0 is not a valid vehicle ID).
+    <br><b>Note:</b> Strictly speaking MAVLink supports 255 concurrent <em>systems</em>, and these can include a mix of vehicles, GCS, antenna trackers and other hardware.</dd>
+
+  <dt>Where can I use MAVLink?</dt>
+  <dd>MAVLink has been shown to work on multiple microcontrollers and operating systems, including ARM7, ATMega, dsPic, STM32 and Windows, Linux, MacOS, Android and iOS.</dd>
+
+  <dt>How safe is MAVLink?</dt>
+  <dd>Very. MAVLink has been used since 2009 to communicate between many different vehicles, ground stations (and other nodes) over varied and challenging communication channels (high latency/noise). It provides methods for detecting packet drops, and the well-established ITU X.25 checksum for packet corruption detection. MAVLink 2, further adds authentication using <a href="../guide/message_signing.md">message signing</a>.</dd>
+</dl>
+
+
+## Developers
+
+<dl>
+  <dt>Can I use MAVLink in a closed source application without copyright issues?</dt>
+  <dd>Yes, without any limitations. The LGPL license only requires you to contribute back changes to <em>MAVLink itself</em>, not any code that uses MAVLink. So you are not required to publish any of your application code.</dd>
+
+  <dt>How does MAVLink detect and decode messages in the byte stream?</dt>
+  <dd>MAVLink waits for the packet start sign, then reads the packet length and matches the checksum after n bytes. If the checksum matches, it returns the decoded packet and waits again for the start sign. If bytes are altered or lost, it will drop the current message and continue the next try on the following message.</dd>
```

Comment:
This text is confusing: " it will drop the current message and continue the next try on the following message."

What does that mean - that it will just move onto the next message?  What do we need to say about resending of packets? 

What I THINK happens is that it will try handle the next message. If the sequence number is wrong it will reject that new message and ask for the previous message to be resent. Is that correct? If not, what happens?